.travis.yml

```diff
@@ -9,8 +10,13 @@ before_script:
   - Rscript -e "install.packages('knitr', repos='http://cran.cnr.Berkeley.edu/')"
   - Rscript -e "install.packages('rmarkdown', repos='http://cran.cnr.Berkeley.edu/')"
 
+before_script:
+  - Rscript -e "install.packages('knitr')"		 +  - Rscript -e "install.packages('knitr',repos='http://cran.cnr.Berkeley.edu/')"
+  - Rscript -e "install.packages('rmarkdown')"		 +  - Rscript -e "install.packages('rmarkdown',repos='http://cran.cnr.Berkeley.edu/')"
+  
```

Comment:
Yeah, I'm not certain how that got in there, I thought I had reverted travis.yml to the version in McDowellLab/loadflex but that doesn't look like what happened.